src/modules/navigation/hooks/helpers.ts

```diff
@@ -68,6 +75,8 @@ export const computeApp = (type: string): string => {
     case 'note':
     case 'public-note-same-instance':
       return 'notes'
+    case 'docs':
+      return 'docs'
```

Comment:
What is this one ?

Reply:
This function is used to determine the slug of the app to open the file. So the `return docs` is the slug, and the `case docs` is the value computed by the [computeFileType](https://github.com/cozy/cozy-drive/blob/88de004bfaf57d87c2d84f696d4b1c1a83053ff4/src/modules/navigation/hooks/helpers.ts#L26) method that returns a string describing the "type" of the io.cozy.files, i.e. if it's 'file', 'directory', 'onlyoffice', etc.
Here it is simply seen as "docs"